dawn/main/ansible/LCM.js

```diff
@@ -0,0 +1,119 @@
+import { ipcMain } from 'electron';
+import LCM from '../../renderer/utils/lcm_ws_bridge';
+import { updateTimer,
+        updateHeart,
+        updateRobot,
+        updateMatch }
+        from '../../renderer/actions/FieldActions';
+import RendererBridge from '../RendererBridge';
+
+class LCMInternals {
+  constructor(stationNumber, bridgeAddress) {
+    this.lcm = null;
+    this.lcmReady = false;
+    this.queuedPublish = null;
+    this.stationNumber = stationNumber;
+    this.bridgeAddress = bridgeAddress;
+    console.log(`LCM SN:${this.stationNumber} BA:${this.bridgeAddress}`);
```

Comment:
This `console.log` (as well as other calls elsewhere) should probably be `logging.log` instead.